mineral-ui/mineral-ui
# WIP: [Link] component

### Description
Added a Link component. This really just a wrapper around an `<a>` since we don't want to re-invent the wheel re: keyboard events, a11y, etc.

closes #149 

### Motivation and context
We decided that Link needs to be a separate component than Button, since a button is a form element, while an anchor is not.

### How to test
1. `npm run build`
2. navigate to Site > Link
3. Play with the links and make sure all the normal props are passed thru
4. It's pretty much just an anchor tag



### Types of changes
- New feature

### Checklist




### How does this PR make you feel?
![hypertext is awesome](https://media.giphy.com/media/3o6ZsX6jbTFoyV5eSc/giphy.gif)
